metaflow/plugins/aws/aws_utils.py

```diff
@@ -109,6 +109,11 @@ def compute_resource_attributes(decos, compute_deco, resource_defaults):
                 # We use the non None value if there is only one or the larger value
                 # if they are both non None. Note this considers "" to be equivalent to
                 # the value zero.
+                #
+                # Skip attributes that are not supported by the decorator.
+                if k not in [*resource_defaults.keys(), *deco.attributes.keys()]:
```

Comment:
nit: this will be recomputed everytime. May be better to make it a set initially and then compare for inclusion. Here for every k, we will recompute the full list.

This also doesn't seem to do what the comment says. My understanding was that it would only let through attributes that are known to, for example, the batch decorator (that's the resource_defaults part) whereas here it lets through anything that is known to the batch_decorator OR the resources decorator which seems to not be exactly what we want (ie: no real different from what it currently is.